<?xml version="1.0"?>
<mavlink>
  <version>3</version>
  <messages>
    <message id="17000" name="JFI">
      <description>J-Fi serial communcation</description>
      <field type="uint8_t" name="tid">topic id</field>
      <field type="uint8_t" name="len">serialized data length</field>
      <field type="uint8_t[250]" name="data">serialized data</field>
    </message>
  </messages>
</mavlink>
